src/Features/VisualBasic/Portable/ExtractInterface/VisualBasicExtractInterfaceService.vb

```diff
@@ -139,7 +133,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExtractInterface
                     Continue For
                 End If
 
-                Dim qualifiedName As QualifiedNameSyntax = SyntaxFactory.QualifiedName(SyntaxFactory.ParseName(implementedInterfaceStatement), SyntaxFactory.IdentifierName(member.Name))
+                Dim qualifiedName As QualifiedNameSyntax = SyntaxFactory.QualifiedName(implementedInterfaceStatementSyntax.GetRightmostName(), SyntaxFactory.IdentifierName(member.Name))
```

Comment:
this seems odd.  how do you know this will be ok?  Seems like the name could end up binding to the wrong thing.  We probably want to just generate the name fully (including 'global.'), but put the Simplificaiton annotation on it.

That said, i'm not going to push hard on this.  Esp as the previous approach was also not good :)